packages/svelte/src/internal/client/dom/elements/misc.js

```diff
@@ -56,3 +57,17 @@ export function add_form_reset_listener() {
 		);
 	}
 }
+
+/**
+ * @param {string} text
+ */
+export function title(text) {
+	render_effect(() => {
+		const previous = document.title;
+		document.title = text;
+
+		return () => {
+			document.title = previous;
```

Comment:
We can also do this, would handle the case described in https://github.com/sveltejs/svelte/pull/14116/files#r1876874053
```suggestion
		const own = {};
		// @ts-expect-error
		document._last_title_setter = own;
		document.title = text;

		return () => {
			// @ts-expect-error
			if (document._last_title_setter === own) {
				document.title = previous;
			}
```

Reply:
Looking at how the code is generated, we should switch it up anyway - right now if your title would contain dynamic content, you would set it two times on an update - once to revert to the previous value, then right away again for the updated value. Wasteful.